receiver/filelogreceiver/README.md

```diff
@@ -179,7 +179,17 @@ Exception in thread 2 "main" java.lang.NullPointerException
 
 ## Offset tracking
 
-`storage` setting allows to define the proper storage extension to be used for storing file offsets. 
+The `storage` setting allows you to define the proper storage extension for storing file offsets.
 While the storage parameter can ensure that log files are consumed accurately, it is possible that
 logs are dropped while moving downstream through other components in the collector.
 For additional resiliency, see [Fault tolerant log collection example](../../examples/fault-tolerant-logs-collection/README.md)
+
+Here is some of the information the `filelog` receiver stores:
+- The number of files it is currently tracking (`knownFiles`).
+- For each file being tracked:
+  - The fingerprint of the file (`Fingerprint.first_bytes`).
```

Comment:
Not sure if we have anywhere else documented the fingerprint logic but maybe you can point to https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/fileconsumer/design.md#fingerprints for those that are interested to learn more about how the fingerprint is calculated and handled.
Otherwise we can move the content here 🤔 ?

Reply:
Yep, good idea, will do.

Reply:
Done in 08405ade31.